english/yuha/yuha.md

```diff
@@ -5,4 +5,6 @@ forked your way out.
  
 But if you want to keep playing (back into the maxtrix) go back to the [beginning] (../marshmallow.md).
 
-Or did they yell ["YOLO"] (./yolo/yolo.md).
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.